client/src/components/Header.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import brutalist from '../themes/brutalist';
+import HeaderTitleSmall from './HeaderTitleSmall.js';
+
+const Container = styled.header`
+  width: 100%;
+  background-color: ${brutalist.colors.backgroundGrey};
+  display: flex;
+  align-content: left;
+  justify-content: space-between;
+  color: ${brutalist.colors.transmitRed};
+`;
+
+const LogoutContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-right: 0.35em;
+  padding-top: 0.15em;
+  cursor: pointer;
+`;
+
+const LogoutText = styled.div`
+  font-family: ${brutalist.fonts.fontRegular};
+  font-size: 0.625rem;
+  margin-top: -2.75em;
+  color: ${(props) =>
+    props.active
+      ? brutalist.colors.transmitRed
+      : brutalist.colors.transmitRedLight};
+`;
+
+function Header({ links, active, activeRotate, onItemClick }) {
+  return (
+    <Container>
+      <HeaderTitleSmall>Transmitter</HeaderTitleSmall>
+      {links.map((link) => (
+        <LogoutContainer
+          key={link.label}
+          active={active === link.label}
+          activeRotate={activeRotate === link.label}
+          onClick={() => onItemClick(link.label)}
+        >
+          <link.Icon
+            active={active === link.label}
+            activeRotate={activeRotate === link.label}
+          />
+          <LogoutText active={active === link.label}>Logout</LogoutText>
+        </LogoutContainer>
+      ))}
+    </Container>
+  );
+}
+
+Header.propTypes = {
+  links: PropTypes.array,
+  active: PropTypes.string,
```

Comment:
`<Header active>` sounds like a boolean. 
I was thinking:
"What is an active header? 
Instead, you want to say which link is active.

```suggestion
  activeLink: PropTypes.string,
```

You might change this again when you add React-Router.

Reply:
✔️